doc/Makefile

```
@@ -81,8 +81,8 @@ gitwash-update:
 #
 
 #SPHINXBUILD="LANG=C sphinx-build"
-NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:10])" 2>/dev/null)
-GITVER ?= $(shell cd ..; $(PYTHON) -c "import versioneer as v; print(v.get_versions()['full-revisionid'][:10])")
+NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:7])" 2>/dev/null)
+GITVER ?= $(shell (cd ..; git rev-parse HEAD | cut -c1-7) || echo Unknown)
```

no big deal, not worth another commit, but is your version of `git rev-parse` missing the [`--short=7` option](https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt---shortlength)
```suggestion
GITVER ?= $(shell (cd ..; git rev-parse --short=7) || echo Unknown)
```

---

I have that option, but it seems to set a lower bound only.